app/assets/stylesheets/getting-started.scss

```diff
@@ -1,7 +1,11 @@
 #hello-there {
-  .avatar {
-    width: 200px;
-    height: 200px;
```

Comment:
This should set the width and height of the new avatar which the user might upload on the getting started page so removing these lines is not an option. The avatar in the well has a default width and height of 50px so we don't have to set that explicitly. (see below)

To have an avatar with the correct size you only need to make sure that this style will only be applied to the uploaded avatar and not to any other avatar on the page.